data/json/recipes/recipe_others.json

```diff
@@ -2263,6 +2263,30 @@
   ]
 },{
   "type" : "recipe",
+  "result": "foldwoodframe",
```

Comment:
For context, you're talking about someone who barely knows how to swing a hammer making something like this:
https://www.google.com/search?q=folding+bicycle&tbm=isch
but out of wood.  This isn't remotely reasonable.